docs/source/dev_guide.rst

```diff
@@ -147,7 +140,24 @@ its own Sphinx documentation and is referenced by the main birdhouse document. P
    .. gitinclude:: https://github.com/Ouranosinc/pavics-sdi.git docs/source/arch/backend.rst
 
    The directive will clone and checkout the repository, then include these external files as if
-   they were part of the native documentation. 
+   they were part of the native documentation.
+
+ .. _writing_tests:
+
+ Writing tests
+ -------------
+
+ Writing test is an essential part of software development. The WPS templates produced by Cookiecutter_ are already including the initial folders for units tests and basic dependencies in the environment.
+ There are two parts of tests:
+
+ * Unit tests:
+ python nose-tests to check the functionality of functions and processes. They are stored in the folder `{bird WPS}/tests` and appropriate test data  `{bird WPS}/tests/testdata`.
```

Comment:
From memory, I think it's pytest and not nose-tests.  Do we use nose in other birds?

Reply:
I don't remember seeing nose anywhere.

Reply:
changed to pytest